k8s101.md

```diff
@@ -654,7 +654,7 @@ Let's build a container image for our worker and push it to our local private re
 $ cd mattermost/worker
 $ export registry="$(kubectl get svc/registry -ojsonpath='{.spec.clusterIP}'):5000"
 $ eval $(minikube docker-env)
-$ docker build -t $registry/mattermost-worker:2.1.0 .
+$ docker build -t $registry/mattermost-worker:5.16.3 .
```

Comment:
Hmm, does this actually work as expected still? I remember @benarent was mentioning that upgrading to newer mattermost version is not trivial and requires some dances around the fire.

Reply:
I tried to update our quickstart, as part of https://github.com/gravitational/quickstart/pull/26  and when I tried to build it, it kept failing. Mattermost has changed a ton since 2.2 to 5.16. https://github.com/gravitational/quickstart/pull/26#issuecomment-517478647

Reply:
you need to apply all that commit, including the Dockerfile change. I tried and it worked.

@benarent are you sure you included all changes in your test?

I'll test everything again though.

Reply:
ow, that's great. Maybe I was updating too many things at one time. Would you be able to double check my Gravity Quickstart works and feel free to merge that into the tutorial  .